lib/beaker/ca_cert_helper.rb

```diff
@@ -0,0 +1,114 @@
+require 'beaker'
+require 'beaker/dsl/helpers/host_helpers'
+
+module Beaker::CaCertHelper
```

Comment:
Do you think this should be require'd from the lib/beaker/puppet_install_helper.rb so that users don't have to add an extra include?